app/src/main/java/com/spartahack/spartahack17/Activity/MainActivity.java

```diff
@@ -97,22 +116,32 @@ protected void onCreate(Bundle savedInstanceState) {
                 item -> {
                     switch (item.getItemId()) {
                         case R.id.action_notifications:
+                            navigateBeforeButton.setVisibility(View.GONE);
+                            settingsIcon.setVisibility(View.GONE);
                             title = getResources().getString(R.string.notifications);
                             addFragment(new AnnouncementFragment());
                             break;
                         case R.id.action_schedule:
+                            navigateBeforeButton.setVisibility(View.GONE);
+                            settingsIcon.setVisibility(View.GONE);
                             title = getResources().getString(R.string.guide);
                             addFragment(new GuideFragment());
                             break;
                         case R.id.action_awards:
+                            navigateBeforeButton.setVisibility(View.GONE);
```

Comment:
i would move these 2 lines into a member function called hideSettigns or something, or make it call regardless of which item was selected